src/common/Constants.h

```diff
@@ -154,6 +154,7 @@ extern const unsigned int SEED_TXN_COLLECTION_TIME_IN_SEC;
 extern const unsigned int TXN_STORAGE_LIMIT;
 
 // Consensus constants
+extern const double TOLERANCE_FRACTION;
```

Comment:
I dont know if it is feasible but can we add some sanity check (cannot be greater than 1 )